src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -2,8 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections;
 using System.Collections.Generic;
 using System.IO;
+using System.Net;
```

Comment:
nit: Is this necessary? If not, remove unused using directive.

Reply:
its not used now, will remove